ui/webui/src/app.jsx

```diff
@@ -72,11 +77,15 @@ export const Application = () => {
                 })}
             </AlertGroup>}
             <AddressContext.Provider value={address}>
-                {!path.length > 0 && <InstallationLanguage />}
+                <ConfContext.Provider value={conf}>
+                    {!path.length > 0 && <InstallationLanguage />}
+                </ConfContext.Provider>
             </AddressContext.Provider>
             {path.length > 0 &&
             <AddressContext.Provider value={address}>
-                <Summary onAddNotification={onAddNotification} />
+                <ConfContext.Provider value={conf}>
+                    <Summary onAddNotification={onAddNotification} />
+                </ConfContext.Provider>
             </AddressContext.Provider>}
```

Comment:
Two questions abut this:
- Are these actually necessary to have in this PR? If the data is not consumed...
- Why use the context provider(s) twice?

Reply:
Correct you can remove it.
It's only necesary when you start consuming this 'data' in another component.  

You need to use the Provider, for each context. You have two different context variables to pass eventually, so in that case to Provider elements.

Reply:
Ah, sorry, I meant, I have two contexts, so two elements, ok. But why do I need to repeat the wraping for each of the actual contents? Can't I have something like...
```
<AddressContext.Provider>
    <ConfContext.Provider>
        {!path.length > 0 && <InstallationLanguage />}
        <Summary onAddNotification={onAddNotification} />
    </ConfContext.Provider>
</AddressContext.Provider>
```
...instead of repeating the wrapping for each of the innermost lines?